applications/nrf_desktop/doc/passkey.rst

```diff
@@ -28,27 +28,30 @@ Configuration
 *************
 
 The passkey input handling is based on ``button_event``.
-You have to enable and configure the :ref:`buttons` module.
 
-Enable the ``passkey`` module with the ``CONFIG_DESKTOP_PASSKEY_BUTTONS`` Kconfig option.
-Define the maximum number of digits in passkey using ``CONFIG_DESKTOP_PASSKEY_MAX_LEN`` option.
+To configure the ``passkey`` module, complete the following steps:
 
-Define the IDs of keys used by the ``passkey`` module in ``passkey_buttons_def.h`` file located in board specific directory in ``configuration`` folder.
-You must define the IDs of keys used to:
+1. Enable and configure the :ref:`buttons` module.
+#. Enable the ``passkey`` module by using the ``CONFIG_DESKTOP_PASSKEY_BUTTONS`` Kconfig option.
+#. Define the maximum number of digits in the passkey by using ``CONFIG_DESKTOP_PASSKEY_MAX_LEN`` option.
+#. Define the IDs of the keys used by the ``passkey`` module in the :file:`passkey_buttons_def.h` file located in the board-specific directory in the :file:`configuration` folder.
+   You must define the IDs of the following keys:
 
-* input the digits,
-* confirm the input,
-* remove last digit (in case the key is held for time longer than 2 seconds, the input is cleared).
+    * Key used to input the digits.
+    * Key used to confirm the input.
+    * Key used to remove the last digit (meaning that if the key is held for longer than 2 seconds, the input is cleared).
```

Comment:
Also there are multiple keys used to confirm input and remove last digit. I will fix it.